src/Microsoft.ML.ImageAnalytics/ImagePixelExtractorTransform.cs

```diff
@@ -160,10 +178,57 @@ public ColInfoEx(Column item, Arguments args)
                 }
             }
 
-            public ColInfoEx(ModelLoadContext ctx)
+            public ColumnInfo(string input, string output, ColorBits colors = ColorBits.Rgb, bool interleave = false)
+                : this(input, output, colors, interleave, false, 1f, 0f)
+            {
+            }
+
+            public ColumnInfo(string input, string output, ColorBits colors = ColorBits.Rgb, bool interleave = false, float scale = 1f, float offset = 0f)
+                : this(input, output, colors, interleave, true, scale, offset)
+            {
+            }
```

Comment:
do you need this constructor?

Reply:
This is for Convert=true case. So yes

---
In reply to: [213860876](https://github.com/dotnet/machinelearning/pull/753#discussion_r213860876) [](ancestors = 213860876)